portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,13 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package com.google.sps.data;
 
-import java.util.*; 
+public final class Comment {
```

Comment:
I know this is a tiny class that doesn't really do anything, but it is a good habit to add comments explaining the purpose for each class.

Could just be as simple as `/** Class to store user's website comments. */`